automation_tools/utils.py

```diff
@@ -85,6 +85,15 @@ def update_packages(*args, **kwargs):
     )
 
 
+def run_command(cmd=None):
+    """ Task to run only sane commands
+    :param str cmd: command to be run
```

Comment:
how do you enforce string in this method? just asking, not a blocker

Reply:
enforcing in python? oh common we all are adults in here, right?

run_command is just sane wrapper around run 
(sane = if cmd is '' or None then bail out)